docs/user_guide/accessibility.md

```diff
@@ -80,71 +93,72 @@ For a simple site (no extra languages or versions), ensure `sphinx-sitemap`
 is installed in your documentation environment, and modify your `conf.py`:
 
 ```python
-    extensions += ["sphinx_sitemap"]
+extensions += ["sphinx_sitemap"]
 
-    html_baseurl = os.environ.get("SPHINX_HTML_BASE_URL", "http://127.0.0.1:8000/")
-    sitemap_locales = [None]
-    sitemap_url_scheme = "{link}"
+html_baseurl = os.environ.get("SPHINX_HTML_BASE_URL", "http://127.0.0.1:8000/")
+sitemap_locales = [None]
+sitemap_url_scheme = "{link}"
 ```
 
 :::
 
 ### Logo best practices
 
-If you use both light and dark themes, it's best to provide a logo that works well in both or to provide an alternative for the dark theme.
-If you have a logo, you can add alt-text to it by adding the following to your
-`conf.py`:
+If you use both light and dark themes, it's best to provide a logo that works
+well in both or to provide an alternative for the dark theme. If you have a
+logo, you can add alt-text to it by adding the following to your `conf.py`:
 
 ```python
-  "logo": {
-      "text": "PyData Theme",
-      "image_dark": "_static/logo-dark.svg",
-      "alt_text": "PyData Theme home",
```

Comment:
I don't know if there really is a way to simplify it because users would encounter the same issue if they were writing HTML. But I guess this is a good example of why I personally dislike config files, because it's a bit hard to convey through the config object itself how its state will affect the state of the end application. 

Anyway, I realized that all of this is covered in much greater detail in a separate page, [Branding and logo](https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/branding.html). So in my last commit I slimmed down this section and linked to that page.